### PR TITLE
Add Popclip

### PR DESCRIPTION
**Pull request by ishu3101**

### CODE REVIEW

File: Casks/popclip.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'popclip' do
```
Comment (vitorgalvao) [inline Casks/popclip.rb]: `cask 'popclip' do`

File: Casks/popclip.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  version '1.5.3'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/popclip.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

File: Casks/popclip.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'popclip' do
+  version '1.5.3'
```
Comment (vitorgalvao) [inline Casks/popclip.rb]: Current is `1.5.5`.

File: Casks/popclip.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  sha256 :no_check
+
+  url "https://d20vhy8jiniubf.cloudfront.net/downloads/PopClip-#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/popclip.rb]: Needs a `# d20vhy8jiniubf.cloudfront.net was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by ishu3101** [comment]
Why do you have to squash the commits?

File: Casks/popclip.rb [hunk 5]
```diff
@@ -0,0 +1,17 @@
+cask 'popclip' do
+  if MacOS.release == :snow_leopard || MacOS.release == :lion || MacOS.release == :mountain_lion
```
Comment (vitorgalvao) [inline Casks/popclip.rb]: Use instead `MacOS.release <= :mountain_lion`.

File: Casks/popclip.rb [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    version '1.5.4'
+    sha256 '8046452efc41988aa20ec0af31bc73aaceba5dd3d68bb4f56dc51959a2966349'
+  elsif MacOS.release >= :mavericks
```
Comment (vitorgalvao) [inline Casks/popclip.rb]: And here just `else`.

**Comment by vitorgalvao** [comment]
Because it makes no sense for us to polute our history with six commits of you just fixing things. A single commit of everything correct is the correct way to go.

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 910fcdb935b61ae6ff205c0eb05aa225022ca4a3. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

File: Casks/popclip.rb [hunk 7]
```diff
@@ -0,0 +1,20 @@
+cask 'popclip' do
+  if MacOS.release <= :mountain_lion
+    version '1.5.4'
+    sha256 '8046452efc41988aa20ec0af31bc73aaceba5dd3d68bb4f56dc51959a2966349'
+  elsif MacOS.release >= :mavericks
+    version '1.5.5'
+    sha256 'e028c21c93c6632c85b63c5ea9b365ae76dd375455baec804506f0091b93a4be'
+  else
+    # Unusual case: there is no fall-through.  The software will stop
```
Comment (vitorgalvao) [inline Casks/popclip.rb]: This is incorrect and makes no sense. `else` will never fire with such a configuration. [This](https://github.com/caskroom/homebrew-cask/blob/61aa694ab1bb4303d27ee2d3a39bb265d9546e01/Casks/popclip.rb#L2L8) is the correct setup.

**Pull request closed without merging**